lib/deqm_test_kit/data_requirements.rb

```diff
@@ -83,6 +86,39 @@ class DataRequirements < Inferno::TestGroup
                "Client data-requirements contains unexpected data requirements for measure #{measure_id}: #{diff}")
       end
     end
+
+    test do
+      title 'Check data requirements returns 400 for missing parameters'
+      id 'data-requirements-02'
+      description 'Data requirements returns 400 when periodStart and periodEnd parameters are omitted'
+
+      run do
+        # Run our data requirements operation on the test client server
+        fhir_operation('Measure/TEST_ID/$data-requirements', body: PARAMS)
```

Comment:
Instead of `TEST_ID`, I think it would be better to use `measure_id` so that there is only one issue with this test case (missing params) and not two with the addition of a technically invalid id